nvflare/apis/impl/seq_relay_manager.py

```diff
@@ -86,7 +86,7 @@ def check_task_send(self, client_task: ClientTask, fl_ctx: FLContext) -> TaskChe
         for i in range(win_start_idx, win_end_idx):
             if client_name == task.targets[i]:
                 # this client is in the window!
-                self.logger.debug(f"last_send_idx={i}")
+                self.logger.debug("last_send_idx={}".format(i))
```

Comment:
This is not an issue (just a mention) but I don't think it's using walrus operator (just a normal f string)

Reply:
yeah it is an f-string, just change it to use format if we want to support 3.5 later.